src/databricks/labs/ucx/install.py

```diff
@@ -285,12 +287,48 @@ def warehouse_type(_):
             groups_config_args["selected"] = [x.strip() for x in selected_groups.split(",")]
         else:
             groups_config_args["auto"] = True
+
+        # Checking for external HMS
```

Comment:
can you put this additional logic into separate private method, that returns spark conf and instance profile?

Reply:
Sure